hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -329,10 +341,12 @@ private function up1402_removeUnusedCaseTypes() {
    * Creates/Updates default case types to be shipped
    * with this extension
    *
+   * WARNING: Resets activity types for an assignment, discarding customization
```

Comment:
It's just a warning that if the client has customized the workflow then this customization will be discarded by calling this method. When I saw it first I thought "oh nice, an easy way to add new case types" but didn't realize it will completely reset all of them.